sftpgo.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: sftpgo
+  version: 2.6.2
+  epoch: 0
+  description: "Full-featured and highly configurable SFTP, HTTP/S, FTP/S and WebDAV server - S3, Google Cloud Storage, Azure Blob"
+  copyright:
+    - license: AGPL-3.0-only
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/drakkan/sftpgo
+      tag: v${{package.version}}
+      expected-commit: 636a1c2c385a16966842da4acad5acd36163ffb4
+
+  - runs: |
+      # Modify the default configuration file
+      sed -i 's|"users_base_dir": "",|"users_base_dir": "/srv/sftpgo/data",|' sftpgo.json && \
+      sed -i 's|"backups"|"/srv/sftpgo/backups"|' sftpgo.json && \
+      sed -i 's|"sqlite"|"bolt"|' sftpgo.json
+
```

Comment:
Note to self compare what official dockerhub sftgo image ships, such that this build remains a drop-in substitute.

Reply:
yeah, we tried to make it drop-in replacement if possible.